financial-connections/src/main/java/com/stripe/android/financialconnections/model/FinancialConnectionsGenericInfoScreen.kt

```diff
@@ -0,0 +1,144 @@
+import android.os.Parcelable
+import com.stripe.android.financialconnections.model.Image
+import com.stripe.android.financialconnections.model.serializer.BodyEntrySerializer
+import kotlinx.parcelize.Parcelize
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Parcelize
+@Serializable
+internal data class FinancialConnectionsGenericInfoScreen(
+    val id: String,
+    val header: Header? = null,
+    val body: Body? = null,
+    val footer: Footer? = null,
+    val options: Options? = null
+) : Parcelable {
+
+    @Parcelize
+    @Serializable
+    internal data class Header(
+        val title: String? = null,
+        val subtitle: String? = null,
+        val icon: Image? = null,
+        val alignment: Alignment? = null
+    ) : Parcelable
+
+    @Parcelize
+    @Serializable
+    internal data class Body(
+        val entries: List<BodyEntry>
+    ) : Parcelable {
+
+        @Serializable(with = BodyEntrySerializer::class)
+        sealed class BodyEntry : Parcelable {
```

Comment:
Nit: Can this just be `Entry`?

Reply:
https://github.com/stripe/stripe-android/pull/8721/commits/e5e39121934c9e056694aa00f5c8bb274fe55ccb